### ISSUE TITLE
DMP 2024 Project Listing

### ISSUE THREAD

**Issue by Shruti3004** [Code4GovTech/server#32]
**Requirement:**

1. As DMP 2024 is starting, tickets from different organizations will be listed on the C4GT website.
2. Currently, there are only community tickets listed on the website with the label "C4GT Community" and the DMP tickets that will be created on GitHub will have the "DMP 2024" label and need to be listed in a separate issue tickets table on the website
3. The labels are not currently being tracked for listing on website, so the requirement is to create another table on the BE to store all the tickets with label "DMP 2024".
4. Once this is done, these tickets can be listed on the website.

**Acceptance criteria:**

- [x] Create another table on supabase to store DMP 2024 labelled tickets
- [x] Update the ticket template for organizations to add issues as shared by @Shruti3004 
- [x] Track the organizations mentioned on tickets
  - [x] Don't fuzzy match name of organisation 

**Reason:**
All this is in preparation for DMP 2024

**Comment by karntrehan** [comment]
Tested by @Shruti3004. Working as expected. Closing now.
